components/atomic/Button/IconButton/index.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from "prop-types";
+import * as Styled from "./styles";
+import { IconComposer, ScreenreaderText } from "@rubin-epo/epo-react-lib";
+
+const IconButton = ({
+  accessibleText,
+  visibleText,
+  isBlock,
+  icon,
+  size,
+  onClickCallback,
+  buttonProps,
+  iconProps,
+  className,
+}) => {
```

Comment:
created an IconButton component that both the WidgetPreview and WidgetSection use to associate icons with their headers